frb_codegen/src/ir/ty.rs

```diff
@@ -70,6 +70,8 @@ impl IrType {
     pub fn as_primitive(&self) -> Option<&IrTypePrimitive> {
         match self {
             Primitive(repr) | Delegate(IrTypeDelegate::PrimitiveEnum { repr, .. }) => Some(repr),
+            #[cfg(feature = "chrono")]
+            Delegate(IrTypeDelegate::Time(_)) => Some(&IrTypePrimitive::I64),
```

Comment:
uber-nit: So, if Time is an i64, are things like Date DateTime etc be primitives?

Reply:
they’re primitives under the hood, iirc it puts seconds & nanoseconds in the I64. This was already done before

Reply:
I see. So I wonder why we treat `Time` and Date/DateTime differently, and is it possible we somehow do the three things in a symmetric/uniform way?

Reply:
My uneducated guess is Timezone, it must be added to DateTimes

Reply:
Oh I see, so you mean Date/DateTime is not a primitive, then I understand it

Reply:
just checked, it’s not a primitive but distuingished by an enum to keep track of type/offset:
```rust
#[cfg(feature = "chrono")]
#[derive(Debug, Clone, Hash, Eq, PartialEq)]
pub enum IrTypeTime {
    Local,
    Utc,
    Naive,
    Duration,
}
```